src/webgl/interactions/InteractionManager.ts

```diff
@@ -0,0 +1,34 @@
+import { ObjectManager } from '../objects'
+import { Container } from 'pixi.js'
+import ApplicationEvents from './ApplicationEvents'
+import RendererOptions from '../RendererOptions'
+import Decelerate from './plugins/Decelerate'
+import HitArea from './hitArea/HitArea'
+import Drag from './plugins/Drag'
+import Zoom from './plugins/Zoom'
+
+export default class InteractionManager {
```

Comment:
### InteractionManager
- primarily for organization and usability

this:
```typescript
this.renderer.dragInteraction.dragging
this.renderer.zoomInteraction.zooming
this.renderer.interactionObjectManager.mount(this.hitArea)
this.renderer.interaction
```

becomes this when you share the reference instead of the entire renderer:
```typescript
this.interactions.isDragging
this.interactions.isZooming
this.interactions.manager.mount(this.hitArea)
this.interactions.container
```